polyfill/lib/time.mjs

```diff
@@ -1,32 +1,30 @@
 /* global __debug__ */
 
+import { GetISO8601Calendar } from './calendar.mjs';
 import { ES } from './ecmascript.mjs';
 import { DateTimeFormat } from './intl.mjs';
 import { GetIntrinsic, MakeIntrinsicClass } from './intrinsicclass.mjs';
 
 import {
-  ISO_YEAR,
-  ISO_MONTH,
-  ISO_DAY,
```

Comment:
I think these three lines still need to be in here.

Reply:
I removed these as part of one of my refactors, but then undid it to make the patch cleaner. This must've slipped through.